### DISCUSSION TITLE
Welcome to pw3270 Discussions!

### DISCUSSION THREAD

**Discussion by PerryWerneck** [PerryWerneck/pw3270#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clebertonello** [comment]
Hello, my name is Cleber.

I already have PW3270 installed. Is there a package available in python for direct communication with the host? I have several routine tasks to perform in this terminal and would like to automate them.

**Reply by PerryWerneck** [reply]
Yes, there is a python bind in https://github.com/PerryWerneck/python3-tn3270

**Comment by Pireddas** [comment]
Subject: Issue with screen rendering in pw3270

Hi Perry,

I’m testing pw3270 on Windows via MSYS2 and connecting to a Hercules/TK5 MVS 3.8j system. The connection works fine, but I’m having a problem with the screen rendering:

The text appears stretched horizontally, occupying the full width of the window.

Vertically, the lines are very compressed, as if the height of the text is only 20–25% of the expected size.

Changing the font to monospace or adjusting window size doesn’t fix it.

I’ve tried adjusting font settings in the hosts.xml file, but the problem persists.

Do you have any guidance on what might be causing this issue or how to fix the screen rendering so that the lines have proper vertical spacing?

Thank you very much for your help!

Best regards,
Ralf

<img width="2559" height="1079" alt="image" src="https://github.com/user-attachments/assets/ef2d9c63-9cbc-4347-b492-6a9650e82977" />

**Comment by PerryWerneck** [comment]
Looks like a font problem, did you tried to change the font in the settings?
